front/app/modules/commercial/native_surveys/messages.ts

```diff
@@ -0,0 +1,13 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  takeTheSurvey: {
+    id: 'app.modules.nativeSurveys.takeTheSurvey',
+    defaultMessage: 'Take the survey',
+  },
+  xSurveys: {
```

Comment:
Suggestion: We could use a more descriptive name that is easier to track even in crowdin